source/crud/crud.rst

```diff
@@ -670,6 +670,25 @@ The OP_QUERY wire protocol only contains a numberToReturn value which drivers mu
 
 Because of this anomaly in the wire protocol, it is up to the driver to enforce the user-specified limit. Each driver MUST keep track of how many documents have been iterated and stop iterating once the limit has been reached. When the limit has been reached, if the cursor is still open, a driver MUST kill the cursor.
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Database-level aggregation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  interface Db {
+
+    /**
+     * Runs an aggregation framework pipeline on the database for pipeline stages
+     * that do not require an underlying collection, such as $currentOp and $listLocalSessions.
+     *
+     * Note: result iteration should be backed by a cursor. Depending on the implementation,
+     * the cursor may back the returned Iterable instance or an iterator that it produces.
+     *
+     * @see https://docs.mongodb.com/manual/reference/method/db.aggregate/
```

Comment:
This should continue to point to the [command documentation](https://docs.mongodb.com/manual/reference/command/aggregate/), rather than this link which is to the mongo shell documentation (let's _definitely_ not use the shell as reference 😄). The command documentation gives guidance around passing `1` in as the value for `aggregate` for "collection agnostic commands"